src/test/scala/org/broadinstitute/dsde/workbench/sam/service/TosServiceSpec.scala

```diff
@@ -105,8 +105,6 @@ class TosServiceSpec extends AnyFlatSpec with TestSupport with BeforeAndAfterAll
     assert(rejectTosStatusResult.get, s"rejectTosStatus(${defaultUser.id}) should reject the tos for the user")
     val getTosStatusResultRejected = tosServiceEnabled.getTosStatus(defaultUser.id).unsafeRunSync()
     assertResult(expected = false, s"getTosStatus(${defaultUser.id}) should have returned false")(actual = getTosStatusResultRejected.get)
-    val isEnabledLdap = regDAO.isEnabled(defaultUser.id, samRequestContext).unsafeRunSync()
-    assertResult(expected = false, "regDAO.isEnabled should have returned false")(actual = isEnabledLdap)
```

Comment:
How did this pass before if it was at the wrong level?

Reply:
Users are disabled by default so this was just asserting the default, `rejectTosStatus` didn't change the enabled status in the `registrationDAO`